File: CovidChartsAndQuiz/app/src/main/java/com/example/myapplication/CovidDeathsChartActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.anychart.AnyChart;
import com.anychart.AnyChartView;
import com.anychart.chart.common.dataentry.DataEntry;
import com.anychart.chart.common.dataentry.ValueDataEntry;
import com.anychart.charts.Pie;

import java.util.ArrayList;
import java.util.List;

public class CovidDeathsChartActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_covid_deaths_chart);

        Pie pie = AnyChart.pie();

        List<DataEntry> data = new ArrayList<>();
        data.add(new ValueDataEntry("Poland", 62734));
        data.add(new ValueDataEntry("USA", 568000));
        data.add(new ValueDataEntry("India", 183000));
        data.add(new ValueDataEntry("Brazil", 378000));
        data.add(new ValueDataEntry("France", 102000));

        pie.data(data);

        AnyChartView anyChartView = (AnyChartView) findViewById(R.id.any_chart_view);
        anyChartView.setChart(pie);
    }
}

File: CovidChartsAndQuiz-Kotlin/app/src/main/java/com/example/zadanie4/CovidDeathsChartActivity.kt
package com.example.zadanie4

import android.R.layout
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.anychart.AnyChart
import com.anychart.AnyChartView
import com.anychart.chart.common.dataentry.DataEntry
import com.anychart.chart.common.dataentry.ValueDataEntry


class CovidDeathsChartActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_covid_deaths_chart)

        val pie = AnyChart.pie()

        val data: MutableList<DataEntry> = ArrayList()
        data.add(ValueDataEntry("Poland", 62734))
        data.add(ValueDataEntry("USA", 568000))
        data.add(ValueDataEntry("India", 183000))
        data.add(ValueDataEntry("Brazil", 378000))
        data.add(ValueDataEntry("France", 102000))

        pie.data(data)

        val anyChartView = findViewById<View>(R.id.any_chart_view) as AnyChartView
        anyChartView.setChart(pie)
    }
}